main.cpp

```diff
@@ -402,6 +498,62 @@ bool Computer::connectPeripherals(std::string peripheralName)
 int main()
 {
     Example::main();
+
+    // 1.
+    Dishwasher dishwasher;
+    dishwasher.washDishes(250);
+    dishwasher.lockDoor();
+    dishwasher.informUser();
+    // 2.
+    VaccumCleaner v;
+
+    std::cout << "value of weight : " << v.weight;
+    std::cout << "value of noiseLevel : " << v.noiseLevel;
+
+    v.collectDust(2000, 100);
+    std::string message = v.informUserWhenFull();
+    std::cout << "message " << message << std::endl;
+    v.retractCord(200);
+    // 3.
+    Bicycle b;
+    b.cycleAtSpeed(120);
+    b.useBreak();
+    b.turn("right");
+    // 4.
+    Truck t;
+    t.moveFood(300);
+    t.drive(140, 20);
+    t.fitTrailer("long", 120);
+    // 5.
+    Keyboard k;
+    k.connectToMotherBoard();
+    k.encodeASCII("X");
+    k.sendCharacterSignal();
+    // 6.
+    Display d;
+    d.regulateBrightness(120);
+    d.displayText("Hello world");
+    d.connectToMotherBoard();
+    // 7.
+    Battery bt;
+    bt.charge(100);
+    bt.discharge(200);
+    bt.monitorCapacity(true);
+    // 8.
+    OperatingSystem os;
+    os.runApplication("OSX");
+    os.installApplication("xcode");
+    os.uninstallApplication("xcode");
+    // 9.
+    Memory m;
+    m.loadData(200);
+    m.offloadData(200);
+    m.accessMemoryBlock(25);
+    // 10.
+    Computer c;
+    c.compileCode(200);
+    c.runProductivityApp("windows", "winzip");
+    c.connectPeripherals("mouse");
```

Comment:
Name things what they are or what they do. 
single letter names are terrible when it comes to helping the reader understand what the purpose of the named object is.
- [ ] revise